doc/nrf/ug_thread_certification.rst

```diff
@@ -69,6 +69,9 @@ Complete the following steps to run the certification tests:
       The overlay file selects the precompiled OpenThread libraries by default.
       It also enables :ref:`multiprotocol support <ug_multiprotocol_support>` with Bluetooth LE advertising.
 
+   .. note::
+      The nRF53 Series devices do not support multiprotocol yet. For those devices please use the file named ``ocerlay-cert-nrf53.conf``.
+
```

Comment:
We should not have two consecutive notes.

```suggestion
      The overlay file selects the precompiled OpenThread libraries.
      
      For nRF52 Series devices, use :file:`harness/overlay-cert.conf`.
      This file also enables :ref:`multiprotocol support <ug_multiprotocol_support>` with Bluetooth LE advertising.

      For nRF53 Series devices, use :file:`harness/overlay-cert-nrf53.conf`.
      This file does not enable multiprotocol support, because there is no multiprotocol support for nRF53 Series devices yet.

```

Reply:
Thanks, changed.